.github/workflows/ci-main.yml

```diff
@@ -83,6 +83,11 @@ jobs:
           - 3306:3306
 
       # MariaDb backend server.
+      # docker run -d --name mariadb \
+      # -p 3307:3306 \
+      # -e MYSQL_DATABASE=test \
+      # -e MYSQL_ROOT_PASSWORD=12345678 \
+      # mariadb:10.4
       mariadb:
         image: mariadb:10.4
```

Comment:
At the time of release, the latest long-term release is [MariaDB 11.4](https://downloads.mariadb.org/mariadb/11.4.4/), which is [maintained for five years](https://mariadb.org/about/#maintenance-policy).

Reply:
done